uaclient/entitlements/base.py

```diff
@@ -38,6 +38,9 @@ class UAEntitlement(metaclass=abc.ABCMeta):
     # Wheter that entitlement is in beta stage
     is_beta = False
 
+    # Help information for the service
+    help_info = None  # type: str
```

Comment:
I think we should make this a class @property as we likely will need to source slightly more dynamic content from either a help JSON file on disk or potentially ua-contracts machine-token response that could be updated dynamically in the future. What we would lke is one simple place to update this help content if desired (it could even live in /var/lib/ubuntu-advantage/help.json). This way we don't have any significant binary changes if we have to update help text, or dynamically overwrite that help content if ua-contracts machine-token response starts providing us dynamic help content.

Reply:
Makes total sense. I will update the code